Multiplatform/Account/LoginQuickConnectView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  LoginQuickConnectView.swift
+//  AmpFin
+//
+//  Created by Daniel Cuevas on 9/27/24.
+//
+
+import SwiftUI
+import AmpFinKit
+
+internal struct LoginQuickConnectView: View {
+    let dismiss: (() -> Void)
+    @State var code: String?
```

Comment:
State variables should be private

Reply:
Resolved in 97739d2d51247b2a6c8f3656f881796c3a829d8c